.gitignore

```diff
@@ -20,3 +20,6 @@ docs/_modules
 .vscode/
 **/pip-wheel-metadata
 
+# IDE specific
```

Comment:
This is not related to the PR. Can you remove these changes?

Reply:
Done